uwblueprint/llsc
# [LLSC-20] Implement Skeleton Email Service

## Notion ticket link

[Setup Basic Email System](https://www.notion.so/uwblueprintexecs/Setup-Basic-Email-System-11110f3fb1dc80a48e88ce63228b926c?pvs=4)



## Implementation description
Setup a quick skeleton for email service with interfaces and classes defined. Will be implementing actual functionality in next pr. Created a test endpoint to ensure that implementation of skeleton classes is done properly just need to add actual logic.



## Steps to test
1. Can run backend properly 
2. Hit the new test email API. Can check fast API docs for params/requirements


## What should reviewers focus on?
* Interface definitions other functionalities that we may ned


## Checklist
